### DISCUSSION TITLE
Instructions to run adapter and other nifi processes

### DISCUSSION THREAD

**Discussion by GitVivekHub** [Sunbird-cQube/community#787] [Q&A]
im looking for instructions to run adapter and other nifi processes/processor groups

**Comment by RNarmatha** [comment]
There are two ways the adaptor s 
1. Through [postman](https://drive.google.com/drive/folders/12Wn7UIHgUhq6U3GzlRN-zdrJMnj1hrPO?usp=sharing) collection 
2. Port forwarding the NIFI
          ssh -L 8096:localhost:8096 -i (pem key) -o ServerAliveInterval=60 (Username) @(Add your IP address)
